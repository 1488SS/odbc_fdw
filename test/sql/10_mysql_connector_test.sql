CREATE EXTENSION odbc_fdw;

CREATE SERVER mysql_aws
FOREIGN DATA WRAPPER odbc_fdw
OPTIONS (
  odbc_DRIVER 'MySQL',
  odbc_SERVER '<%= mysql_server %>',
  odbc_DATABASE '<%= mysql_database %>',
  encoding '<%= mysql_server %>'
);

CREATE USER MAPPING FOR postgres SERVER mysql_aws
OPTIONS (
  odbc_UID '<%= mysql_username %>',
  odbc_PWD '<%= mysql_password %>'
);

IMPORT FOREIGN SCHEMA fdw_tests
FROM SERVER mysql_aws
INTO public;

SELECT * FROM odbc_test_table;