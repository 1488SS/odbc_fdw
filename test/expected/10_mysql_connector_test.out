SELECT * FROM odbc_test_table;
 id |   name    |    description    | users |       createdtime        
----+-----------+-------------------+-------+--------------------------
  1 | test name | sdklfjslkdfjlskdf |   100 | Fri Aug 12 00:00:00 2016
(1 row)
